requirements.txt

```diff
@@ -13,9 +13,10 @@ argparse==1.2.1
 watchdog==0.8.3
 peakutils==1.0.3
 treeherder-client==3.1.0
+matplotlib>=1.5.3
+pandas>=0.19.2
```

Comment:
Personally, I agree your suggestion in short term or for specific releases. Thanks.